pxc/jenkins/pxc-package-testing.groovy

```diff
@@ -13,11 +13,22 @@ void installDependencies() {
         python3 -m pip install setuptools wheel
```

Comment:
For master I believe we will have only this groovy file. Please apply previous comments also here.